tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -224,7 +224,7 @@ public static class Tags
         /// <summary>
         /// The raw command sent to Redis.
         /// </summary>
-        internal const string RedisRawCommand = "redis.raw_command";
+        internal const string RedisRawCommand = "meta.db.statement";
```

Comment:
I apologize for interrupting your Draft PR but:
```suggestion
        internal const string RedisRawCommand = "db.statement";
```

Reply:
Fixed in e6bd2b2abd6d24cc1eba151569e6defdde6fda13